website/docs/r/ec2_traffic_mirror_target.html.markdown

```diff
@@ -25,6 +25,11 @@ resource "aws_ec2_traffic_mirror_target" "eni" {
   description          = "ENI target"
   network_interface_id = aws_instance.test.primary_network_interface_id
 }
+
+resource "aws_ec2_traffic_mirror_target" "gwlb" {
+  description                       = "GWLB target"
+  gateway_load_balancer_endpoint_id = aws_lb.lb.arn
```

Comment:
Since there is also an example above that uses `aws_lb.lb.arn` for `network_load_balancer_arn` perhaps we can change the `lb` to `gwlb` to avoid confusion between the two

```suggestion
  gateway_load_balancer_endpoint_id = aws_lb.gwlb.arn